packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1204,13 +1211,14 @@ class WidgetsApp extends StatefulWidget {
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
   static bool get debugShowWidgetInspectorOverride {
-    return _debugShowWidgetInspectorOverrideNotifier.value;
+    return _getDebugShowWidgetInspectorOverrideNotifier.value;
   }
   static set debugShowWidgetInspectorOverride(bool value) {
-    _debugShowWidgetInspectorOverrideNotifier.value = value;
+    _getDebugShowWidgetInspectorOverrideNotifier.value = value;
   }
 
-  static final ValueNotifier<bool> _debugShowWidgetInspectorOverrideNotifier = ValueNotifier<bool>(false);
+  static ValueNotifier<bool>? _debugShowWidgetInspectorOverrideNotifier;
+  static final ValueNotifier<bool> _getDebugShowWidgetInspectorOverrideNotifier = _debugShowWidgetInspectorOverrideNotifier ?? ValueNotifier<bool>(false);
```

Comment:
This needs to be
```
static final ValueNotifier<bool> get _debugShowWidgetInspectorOverrideNotifier =>_debugShowWidgetInspectorOverrideNotifierCache ??= ValueNotifier<bool>(false);
```
Otherwise 
_getDebugShowWidgetInspectorOverrideNotifier and _debugShowWidgetInspectorOverrideNotifier will get out of sync after 
_debugShowWidgetInspectorOverrideNotifier is nulled out.

it is not idiomatic Dart to have fields or getters that start with the word get so I would change the new getter to be the standard name and add a prefix to the cached value. Maybe `Cache` on the end or some other suffix more in line with other code in Flutter.